src/Avalonia.Visuals/Platform/IEllipseGeometryImpl.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) The Avalonia Project. All rights reserved.
+// Licensed under the MIT license. See licence.md file in the project root for full license information.
+
+namespace Avalonia.Platform
+{
+    /// <summary>
+    /// Defines the platform-specific interface for a <see cref="Avalonia.Media.EllipseGeometry"/>.
+    /// </summary>
+    public interface IEllipseGeometryImpl : IGeometryImpl
```

Comment:
Do we need these interfaces? I don't see any usage for them, the factory can just return IGeometryImpl for everything.

Reply:
Most likely not, I got that from previous PR and I kept it. Can easily remove it if wanted.

Reply:
Removed interfaces.